end-to-end-tests/.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  extends: 'ndla',
+  rules: {
+    'jsx-a11y/href-no-hash': 1,
+
+    'import/prefer-default-export': 0,
+    'import/no-named-as-default': 0,
+    'no-undef': 0,
+    'no-unused-expressions': 0,
+    "comma-dangle": ["error", {
```

Comment:
I don't think it is necessary repeat rules, globals and extend from the root eslintrc

Reply:
Btw, the comma-dangle rules can probably be removed in the root eslintrc to.